src/firefly/tests/test_grid_scan_window.py

```diff
@@ -59,52 +59,53 @@ async def test_time_calculator(display, sim_registry, ion_chamber):
 
 
 @pytest.mark.asyncio
-async def test_step_size_calculation(display):
+async def test_step_size_calculation(display, qtbot):
     # Set up the display with 2 regions
     await display.update_regions(2)
 
     # Region 0: Set Start, Stop, and Points
     region_0 = display.regions[0]
     region_0.start_line_edit.setText("0")
     region_0.stop_line_edit.setText("10")
-    region_0.scan_pts_spin_box.setValue(5)
+    region_0.scan_pts_spin_box.setValue(12)
+    qtbot.wait(10)
```

Comment:
These ``qtbot.wait()`` calls are causing the CI failures. Not exactly sure why, but if I remove them, the tests pass. If you need them for some other reason, you might be able to use ``await asyncio.sleep(0.01)`` instead.